hcloud/exp/kit/randutil/id.go

```diff
@@ -0,0 +1,20 @@
+package randutil
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+)
+
+// GenerateID returns a hex encoded random string with a len of 8 chars similar to
+// "2873fce7".
+//
+// This function panics if [rand.Read] returns an error.
+func GenerateID() string {
+	b := make([]byte, 4)
+	_, err := rand.Read(b)
+	if err != nil {
+		panic(fmt.Errorf("failed to generate random string: %w", err))
```

Comment:
I am not sure about this one, should I add another function called `MustGenerateID` that does not return an error and so the user can choose which function to use based on the context?